src/platform/site-wide/header/components/LogoRow/index.js

```diff
@@ -30,6 +34,8 @@ export const LogoRow = ({
 
         {/* Mobile menu button */}
         <button
+          aria-controls="header-nav-items"
+          aria-expanded={isMenuOpen ? 'true' : 'false'}
```

Comment:
couldn't you just do aria-expanded={isMenuOpen}

Reply:
I wasn't sure actually haha can I?